### PR TITLE
fix: pool table view ROI calculator mobile issue

### PR DESCRIPTION
**Pull request by Chef-Yogi**

### CODE REVIEW

File: src/views/Pools/components/Apr.tsx [hunk 1]
```diff
@@ -21,9 +21,18 @@ interface AprProps extends FlexProps {
   stakedBalance: BigNumber
   showIcon: boolean
   performanceFee?: number
+  fontSize?: string
+  iconSize?: string
```
Comment (chef-jojo) [inline src/views/Pools/components/Apr.tsx:25]: is it needed?

File: src/views/Pools/components/PoolStatsInfo.tsx [hunk 2]
```diff
@@ -41,8 +43,11 @@ const PoolStatsInfo: React.FC<ExpandedFooterProps> = ({
     vaultKey,
     profileRequirement,
     isFinished,
+    userData: uD,
   } = pool
 
+  const stakedBalance = uD?.stakedBalance ? new BigNumber(uD.stakedBalance) : BIG_ZERO
```
Comment (chef-jojo) [inline src/views/Pools/components/PoolStatsInfo.tsx:49]: it should be BigNumber already

File: src/views/Pools/components/PoolStatsInfo.tsx [hunk 3]
```diff
@@ -41,8 +43,11 @@ const PoolStatsInfo: React.FC<ExpandedFooterProps> = ({
     vaultKey,
     profileRequirement,
     isFinished,
+    userData: uD,
```
Comment (memoyil) [inline src/views/Pools/components/PoolStatsInfo.tsx:46]: We can name it poolUserData

File: src/views/Pools/components/Apr.tsx (hunk 1 above)
Comment (Chef-Yogi) [reply src/views/Pools/components/Apr.tsx:25]: iconSize is not needed

Comment (Chef-Yogi) [reply src/views/Pools/components/Apr.tsx:25]: will remove it

File: src/views/Pools/components/PoolStatsInfo.tsx (hunk 3 above)
Comment (Chef-Yogi) [reply src/views/Pools/components/PoolStatsInfo.tsx:46]: ok thanks!

**Comment by memoyil** [comment]
LGTM 👍

File: src/views/Pools/components/PoolStatsInfo.tsx [hunk 4]
```diff
@@ -1,4 +1,5 @@
 import { Button, Flex, Link, LinkExternal, MetamaskIcon, Skeleton, Text, TimerIcon } from '@pancakeswap/uikit'
+import BigNumber from 'bignumber.js'
```
Comment (memoyil) [inline src/views/Pools/components/PoolStatsInfo.tsx:2]: eslint

**Review by chef-jojo — APPROVED** [review]

**Pull request merged**